web_ui/src/pages/project-details/components/project-models/train-model-dialog/advanced-settings/data-management/data-management.component.tsx

```diff
@@ -27,32 +27,43 @@ const getAugmentationParameters = (configuration: TrainingConfiguration) => {
     return augmentation;
 };
 
-const getTilingParameters = (_configParameters: TrainingConfiguration) => {
-    return undefined;
+const getTilingParameters = (configuration: TrainingConfiguration) => {
+    return configuration.datasetPreparation.augmentation.tiling;
 };
 
 export const DataManagement: FC<DataManagementProps> = ({ trainingConfiguration, onUpdateTrainingConfiguration }) => {
     const tilingParameters = getTilingParameters(trainingConfiguration);
```

Comment:
np: below we do,
```
    const subsetSplitParameters = trainingConfiguration.datasetPreparation.subsetSplit;
```

so I guess we could do the same here?
```suggestion
    const tilingParameters = configuration.datasetPreparation.augmentation.tiling;
```

Reply:
Ok, I'll update in a separate PR (to avoid asking for reviews)